com.unity.render-pipelines.core/CoreRP/Common/XRGraphicsConfig.cs

```
@@ -19,24 +19,22 @@ public class XRGraphicsConfig
         public float viewportScale;
         public bool useOcclusionMesh;
         public float occlusionMaskScale;
-        public bool showDeviceView;
-        public GameViewRenderMode gameViewRenderMode;
-        
+
         public void SetConfig()
         { // If XR is enabled, sets XRSettings from our saved config
             if (!enabled)
-                Assert.IsFalse(enabled);
+                return;
+            Assert.IsTrue(enabled);
```

Shouldn't the assert be before the test?  This way user would be warned before the method silently returns.

---

I will remove the asserts. If I place them before the return, then when VR is disabled, the assertions will fire, causing Katana tests to fail even if stereo is not intended to be enabled for those tests.